src/ui/public/directives/__tests__/fixed_scroll.js

```diff
@@ -9,10 +9,12 @@ import Promise from 'bluebird';
 describe('FixedScroll directive', function () {
 
   let compile;
+  let timeout;
```

Comment:
Our [angular friendly version](https://github.com/elastic/kibana/blob/2f6654bcec1db4f74a673906f5819581b94279ee/src/ui/public/debounce/debounce.js) of debounce actually uses $timeout under the hood so I still need to flush those timeouts to get the tests to work.